extra/make-cfg.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+#  This script will generate config for make-deb and deployment on bintray in TravisCI, 
+#  This script can't work without TravisCI
+
+# Ensure we're always in the right directory.
+SCRIPT_DIR="$( cd "$( dirname "$0" )" && pwd )"
+cd $SCRIPT_DIR/..
+
+#Generate SYMBOL according to branch or tag
+if [ -n "$TRAVIS_BRANCH" ]; then
+   RRANCH=${TRAVIS_BRANCH}
+fi
+
+if [[ "${RRANCH}" == *"master"* ]]; then
+   SYMBOL=devel
+elif [[ "${RRANCH}" == *"release"* ]]; then
+   SYMBOL=rc
+fi
+
+#Tag format is to be confirmed
+if [ -n "$TRAVIS_TAG" ] && [[ "$TRAVIS_TAG" =~ ^[0-9.]+$ ]]; then
+   SYMBOL=release
+fi
+
+# This appends a datestring formatting specifically to be increasing
+# based on the last date of the commit in this branch to provide increasing
+# DCH version numbers for building debian packages for bintray.
+GITCOMMITDATE=$(git show -s --pretty="format:%ci")
+DATESTRING=$(date -d "$GITCOMMITDATE" -u +"%Y%m%d%H%M%SZ")
+
+if [ -z "$DEBFULLNAME" ]; then
+        export DEBFULLNAME=`git log -n 1 --pretty=format:%an`
+fi
+
+if [ -z "$DEBEMAIL" ]; then
+        export DEBEMAIL=`git log -n 1 --pretty=format:%ae`
+fi
+
+#deb name for release is to be confirmed
+if [ -z "$DEBBRANCH" ] && [ "$SYMBOL" != "release" ]; then
+        export DEBBRANCH=`echo "${SYMBOL}-${DATESTRING}" | sed 's/[\/\_]/-/g'`
+fi
+
+if [ -z "$DEBPKGVER" ]; then
+  export DEBPKGVER=`git log -n 1 --pretty=oneline --abbrev-commit`
+fi
+
+if [ -z "$DCHOPTS" ]; then
+        export DCHOPTS="-l ${DEBBRANCH} -u low ${DEBPKGVER}"
```

Comment:
In make-deb.sh it also "export variables", so I think "env vars" is a concise way to pass vars to make-deb.sh.

and make-deb.sh can run stand-alone without any other scripts.